xfce/xfce4-terminal/Portfile

```diff
@@ -3,11 +3,11 @@
 PortSystem 1.0
 
 name            xfce4-terminal
-version         0.8.9.2
+version         1.1.3
 revision        1
```

Comment:
`revision` should be reset to 0 when the `version` is increased.

Reply:
My bad, fixed now.